doc/source/overview/basic.rst

```diff
@@ -2,17 +2,17 @@
 PyAnsys Project Organization
 ############################
 
-The PyAnsys project is hosted on GitHub at `PyAnsys
+The `PyAnsys Project <https://docs.pyansys.com/>`_ is hosted on GitHub at `PyAnsys
 <https://github.com/pyansys>`_. It contains several repositories with 
 Python libraries that interface with Ansys products or services. 
 To try out a library, visit one of these links:
 
-* `PyAnsys Project <https://docs.pyansys.com/>`_
+* * `PyAEDT`_
+* `PyDPF-Core <https://github.com/pyansys/DPF-Core>`_
+* `PyDPF-Post <https://github.com/pyansys/DPF-Post>`_
```

Comment:
```suggestion
* `PyAEDT`_
* `PyDPF-Core <https://github.com/pyansys/DPF-Core>`_
* `PyDPF-Post <https://github.com/pyansys/DPF-Post>`_
```

PyAEDT appears at a second level, I don't know if this was intended... but it doesn't look like